### ISSUE TITLE
Command "phpcs ." doesnt work correctly (bug report)

### ISSUE THREAD

**Issue by Solvent13** [squizlabs/PHP_CodeSniffer#3393]
**Describe the bug**

The command ```phpcs .``` (comma - is the current directory path) first outputs all existing error types and their descriptions in the range from 1 to 18989. And only then it shows the report. However if i set specific path instead comma (e.g. ```phpcs file.php```) then CodeSniffer works correctly.

![issue](https://user-images.githubusercontent.com/46854749/126083268-6d0d96de-e8f1-4fae-b20b-e18515730b83.png)

**Code sample**

<ins>Dockerfile</ins>
```
FROM php:7.4-cli-alpine

ARG UID
ARG GID
ENV PATH="${PATH}:/app/vendor/bin"

COPY common/php/conf.d/security.ini $PHP_INI_DIR/conf.d
COPY development/config-php $PHP_INI_DIR/conf.d

RUN mv $PHP_INI_DIR/php.ini-development $PHP_INI_DIR/php.ini; \
    apk add --no-cache unzip shadow php7-pecl-xdebug git; \
    rm -rf /var/cache/apk/*; \
    curl -sS https://getcomposer.org/installer | php -- --install-dir=/usr/local/bin/ --filename=composer --quiet; \
    chown $UID:$GID /home; \
    usermod -u $UID www-data; \
    groupmod -g $GID www-data

WORKDIR /app
```
<ins>docker-compose.yaml</ins>
```
services:
    back-php-cli:
        build:
            context: backend/docker
            dockerfile: development/php-cli/Dockerfile
            args:
               UID: 1000
               GID: 1000
        volumes:
            - ./backend/src:/app
        container_name: back-php-cli
        tty: true
```

**Custom ruleset**
```xml
<?xml version="1.0"?>
<ruleset name="Configs for PHP_CodeSniffer">

    <config name="php_version" value="70421"/>
    <file>.</file>
    <arg name="colors"/>
    <arg value="sp"/>
    <arg name="parallel" value="50"/>

    <arg name="cache" value="storage/app/php-cs.cache"/>

    <rule ref="PSR12">
        <exclude name="PSR12.Traits.UseDeclaration.MultipleImport"/>
        <exclude name="Generic.Files.LineLength.TooLong"/>
    </rule>

    <rule ref="Internal.NoCodeFound">
        <severity>0</severity>
    </rule>

    <rule ref="Squiz.WhiteSpace.SuperfluousWhitespace">
        <properties>
            <property name="ignoreBlankLines" value="false" />
        </properties>
    </rule>

    <rule ref="PSR1.Methods.CamelCapsMethodName.NotCamelCaps">
        <exclude-pattern>tests</exclude-pattern>
    </rule>

    <exclude-pattern>vendor</exclude-pattern>
    <exclude-pattern>storage</exclude-pattern>
    <exclude-pattern>bootstrap/cache</exclude-pattern>
</ruleset>
```

**To reproduce**

1. Start the Docker container according to the files described above (unnecessary settings can be removed)
2. Use custom ruleset file described above
3. Run ```phpcs``` inside the container (of course, some php files must be in the directory)
```
output: see screenshot above
```

**Expected behavior**

Default behavior (without first displaying all 18,000+ existing error types)

**Versions:**
 - OS: docker image ```php:7.4-cli-alpine```
 - PHP: 7.4.21
 - PHPCS: 3.6.0 (stable)
 - Standard: it specifies PSR12 in the ruleset file (see above), but in fact, other standarts errors are also included in the phpcs checking (for example PSR1, Squiz). The command ``` phpcs -i ``` shows PSR2, Squiz, PEAR, Zend, MySource, PSR12 and PSR1

**Comment by gsherwood** [comment]
I'm not sure what bug you are reporting here. The screenshot shows a pretty normal PHPCS report, showing violations per file. Can you be a bit more specific about what "default behaviour" means to you, because the default output is the per-file report, as per the screenshot.

**Comment by gsherwood** [comment]
One extra bit of information, that number `18989` you have pointed to is not an error number. That is a line number. The error report is showing you the violations for a file with about 19k lines in it. If you scroll to the top, it will tell you what that file is.

Alternatively, run PHPCS using `--report=summary` and it will instead show you a list of files and how many violations are in each.

**Comment by Solvent13** [comment]
You are right. The command phpcs --report=summary clarified the situation. The 6000+ displayed errors did not fit all in the console and it was not clear where they were coming from, so the output looked like a bug.

**Comment by gsherwood** [comment]
Glad to hear that fixed it.
